### DISCUSSION TITLE
[Bug] ?? Problems in Adminer

### DISCUSSION THREAD

**Discussion by kriscr** [pimcore/pimcore#11368] [Q&A / Support] [answered]
When using Adminer, I'm getting an error.
I used COMPOSER_MEMORY_LIMIT=-1 composer create-project pimcore/skeleton my-project to star clean project.
PHP 8.0.14

HTTP 500 Internal Server Error
Warning: Cannot modify header information - headers already sent by (output started at /phpapp/vendor/vrana/adminer/adminer/db.inc.php:107)

Has anyone else having this problem? If so, could you find a fix?

**Comment by blankse** [comment, marked as answer]
@kriscr There is a open issue for this: https://github.com/pimcore/pimcore/issues/11304

**Comment by kriscr** [comment]
Ok, thanks.
